### PR TITLE
Add Raspbian 8 download button [stage-2]

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc 
Please review. If you'd like to test out the 'Add Display' email, you can get it sent by adding a new Display via https://apps-stage-2.risevision.com/
Thx!

File: web/partials/displays/download-player.html [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
+<div class="row">
   <div class="col-sm-6">
-    <a class="btn btn-primary btn-block u_margin-sm-bottom" ng-click="displayTracker('Player Download', display.id, display.name, 'Ubuntu 64bit')" href="http://install-versions.risevision.com/installer-lnx-64.sh" target="_blank">
-      {{'displays-app.download.linux64' |  translate }}
-      <img class="icon-right" src="https://s3.amazonaws.com/Rise-Images/Icons/ubuntu.png" width="16" height="16">
+    <a id="downloadRaspbianARMv7" class="btn btn-primary btn-block u_margin-sm-bottom"
+    ng-click="displayTracker('Player Download', display.id, display.name, 'Raspbian ARMv7')"
+    href="http://install-versions.risevision.com/installer-lnx-armv7l.sh">
+      {{'displays-app.download.armv7l' |  translate }}
+      <img class="icon-right" src="https://s3.amazonaws.com/Rise-Images/Icons/raspbian.png" width="16" height="16">
```
Comment (alex-deaconu) [inline web/partials/displays/download-player.html]: I'm not sure if there's a specific reason for this, but the Linux download links have `target="_blank"` while Windows do not. @RV-Adrian @ezequielc any ideas why this is the case? I'm wondering if it's intentional. Thanks.

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good. Just one question regarding an inconsistency I noticed. Thanks.

**Comment by RV-Adrian** [comment]
@alex-deaconu Good catch. On my side it opens and then immediately closes a tab before the download begins. If @ezequielc isn't aware of a reason, it looks best to remove it. Thx!

**Comment by alex-deaconu** [comment]
@RV-Adrian the only thing I'm worried about is if in some instances the `.sh` file would open as a text file in the browser, instead of download. If that's the case, it's probably best to open it in a new tab as it would redirect the User from Apps otherwise. Can't really confirm that however.

**Comment by RV-Adrian** [comment]
@alex-deaconu 
I tried it without 'target=_blank" in Chrome on OSx and in Win10. It triggered download both times. Should we try it in Ubuntu or other browsers to be certain?

**Comment by ezequielc** [comment]
@alex-deaconu @RV-Adrian I don't remember we adding that intentionally. As @RV-Adrian tested and worked fine without it, I would say we can remove. Thanks

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc 
Sounds good. Removed and re-staged, thanks.

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good, thanks for fixing 👍 . 

Not sure what the conclusion was, whether or not we should deploy this today or next week.

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc I got the vibe from SoS that anything not-critical should wait until next week? I can run by Alan though.

**Comment by ezequielc** [comment]
@RV-Adrian Yes, I think Alan mentioned adding a 'Beta' label or something. You should double check before merging. Thanks

**Comment by RV-Adrian** [comment]
@ezequielc has fixed the build process. Aiming to merge and deploy this morning
@alex-deaconu @fjvallarino FYI, thanks
https://trello.com/c/O1ggvYlu/2216-ux-add-display-email-update-layout-fix-1

**Comment by alex-deaconu** [comment]
@RV-Adrian just FYI, decision in SoS was to hold off on deploying this until the RPi issues are fixed. Thanks.

**Comment by RV-Adrian** [comment]
@alex-deaconu Thanks for the update, will keep the card in Blocked for now

**Comment by alex-deaconu** [comment]
@ezequielc @RV-Adrian confirmed in SoS that we're good to go with releasing this. Merging. Thanks!

**Pull request merged**